src/renderer/themes.css

```diff
@@ -263,6 +263,9 @@ it can be safely elided. This looks quite pleasant on this theme. */
   --logo-text-bar-color: url("../../_icons/textBlackSmall.png");
 }
 
+.mainLightGrey,
+.mainDarkGrey,
```

Comment:
Looks like these 3 lines (266-268) can be removed

Reply:
Removed the excess code you pointed out, as well as the same for the sec greys. My testing revealed that they still worked without them.